src/components/CardBanners/index.tsx

```diff
@@ -0,0 +1,17 @@
+import "./styles.scss"
+import streamBanners from "../../data/images"
+
+
+const StreamCards = () => {
+
+  return (
+    <div className="stream-banners">
+      {streamBanners.map((imageUrl, index) => (
+        <img key={index} src={imageUrl} id={`banner-${index + 1}`} alt={`Image ${index + 1}`} />
+      ))}
+    </div>
```

Comment:
why are these streamBanners at the same time as StreamCards, please be consistent